src/Infrastructure/LeanCode.AuditLogs/ServiceCollectionExtensions.cs

```diff
@@ -0,0 +1,25 @@
+using Microsoft.Extensions.DependencyInjection;
+
+namespace LeanCode.AuditLogs;
+
+public static class ServiceCollectionExtensions
+{
+    public static IServiceCollection AddAuditLogs<TAuditLogStorage>(this IServiceCollection services)
```

Comment:
`AddAuditLogs<AzureBlobAuditLogStorage>` won't work. We should prevent that on API level (i.e. remove this one).